.github/workflows/e2e-tests.yml

```diff
@@ -43,6 +46,8 @@ jobs:
         uses: helm/chart-testing-action@v2.3.0
       - name: Create kind cluster
         uses: helm/kind-action@v1.5.0
+        with:
+          kubectl_version: ${{ matrix.k8s-version }}
```

Comment:
Does it create k8s cluster with this version or just kubectl?

Reply:
My bad, this variable only effects the kubectl version, not cluster, I will check other variables.
@dmitryax

Reply:
Using `node_image` is right.
![image](https://user-images.githubusercontent.com/11855957/220014123-d2efa493-a441-4121-9c8f-4986f805a139.png)